detection-rules/attachment_malicious_onenote_commands.yml

```
@@ -0,0 +1,28 @@
+name: "Attachment: Malicious OneNote Commands"
+description: |
+  Scans for OneNote attachments that contain suspicious commands that may indicate malicious activity.
+references:
+  - "https://www.trustedsec.com/blog/new-attacks-old-tricks-how-onenote-malware-is-evolving/"
+type: "rule"
+severity: "high"
+source: |
+    type.inbound
+    and any(attachments, .file_extension == 'one'
+        and any(beta.binexplode(.),
+            any(.scan.strings.strings,
+                strings.ilike(., 
+                    "*WshShell*",
+                    "*ExecuteCmdAsync*",
+                    "*CreateObject*",
+                    "*Wscript.Shell*",
+                    "*schtasks*",
+                    "*CreateProcess*",
+                    "*winmgmts*",
+                    "*SetEnvironmentVariable*",
```

```suggestion
                    "*CreateProcess*",
                    "*winmgmts*",
                    "*SetEnvironmentVariable*",
                    "*powershell*"
```

@arnydo Thoughts on adding powershell?

---

Absolutely!